packages/amplify_flutter/lib/amplify_hub.dart

```diff
@@ -39,29 +37,28 @@ class AmplifyHub {
     }
 
     controller = StreamController.broadcast(
-      onListen: () {
-        channels.forEach((c) {
-          if (availableStreams[c] != null) {
-            StreamSubscription subscription = availableStreams[c].listen((msg) {
-              /// Emit events via Hub
-              controller.add(msg);
-            });
-            platformSubscriptions.add(subscription);
-          } else {
-            print('You are attempting to listen to a plugin that has not been added.');
-          }
-      });
-    },
-      onCancel: cancelPluginStreams
-    );
+        onListen: () {
+          channels.forEach((c) {
+            if (availableStreams[c] != null) {
+              StreamSubscription subscription =
+                  availableStreams[c].listen((msg) {
+                /// Emit events via Hub
+                controller.add(msg);
+              });
+              platformSubscriptions.add(subscription);
+            } else {
+              print(
+                  'You are attempting to listen to a plugin that has not been added.');
+            }
+          });
+        },
+        onCancel: cancelPluginStreams);
 
     return controller.stream.listen(listener);
   }
 
   /// Adds Plugin level streams in preparation for 'listen'
-  void addChannel(HubChannel name, StreamController controller) async  {
-    if (!Amplify.isConfigured) {
```

Comment:
Why are we removing this line?  It was added as part of @jamesonwilliams 's PR feedback.

Reply:
Discussed offline, `isConfigured` is no longer exposed as a public API from `amplify.dart`